### ISSUE TITLE
Test failure: integrations.clprf.test_clprf.TestSearchIntegration.test_core18

### ISSUE THREAD

**Issue by lintool** [castorini/pyserini#944]
```
% python -m unittest integrations.clprf.test_clprf.TestSearchIntegration.test_core17_rrf
Running core17 topics, saving to ./integrations/tmp3631/core17_lr.txt...

...

Traceback (most recent call last):
  File "/Users/jimmylin/opt/anaconda3/envs/pyserini-dev/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/jimmylin/opt/anaconda3/envs/pyserini-dev/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/jimmylin/workspace/pyserini/pyserini/fusion/__main__.py", line 37, in <module>
    trec_runs = [TrecRun(filepath=path,resort=args.resort) for path in args.runs]
  File "/Users/jimmylin/workspace/pyserini/pyserini/fusion/__main__.py", line 37, in <listcomp>
    trec_runs = [TrecRun(filepath=path,resort=args.resort) for path in args.runs]
  File "/Users/jimmylin/workspace/pyserini/pyserini/trectools/_base.py", line 104, in __init__
    self.read_run(self.filepath,self.resort)
  File "/Users/jimmylin/workspace/pyserini/pyserini/trectools/_base.py", line 113, in read_run
    self.run_data["rank"] = self.run_data.groupby("topic")[["docid","score"]].rank(ascending=False,method='first')
  File "/Users/jimmylin/opt/anaconda3/envs/pyserini-dev/lib/python3.8/site-packages/pandas/core/frame.py", line 3602, in __setitem__
    self._set_item_frame_value(key, value)
  File "/Users/jimmylin/opt/anaconda3/envs/pyserini-dev/lib/python3.8/site-packages/pandas/core/frame.py", line 3729, in _set_item_frame_value
    raise ValueError("Columns must be same length as key")
ValueError: Columns must be same length as key
F
======================================================================
FAIL: test_core17_rrf (integrations.clprf.test_clprf.TestSearchIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jimmylin/workspace/pyserini/integrations/clprf/test_clprf.py", line 271, in test_core17_rrf
    self.assertEqual(status, 0)
AssertionError: 256 != 0

----------------------------------------------------------------------
Ran 1 test in 650.466s

FAILED (failures=1)
```

This might be an issue with the pandas version, as I'm getting this failure on my mac:

```
% pip list | grep pandas
pandas             1.3.5
```

Whereas the test passes fine on `orca`:

```
$ pip list | grep pandas
pandas                  1.1.4
```

@yuki617 would you mind taking a look?

**Comment by yuki617** [comment]
It caused by new version of pandas. Will send a new PR fixing it.

**Comment by lintool** [comment]
Closed by #946

**Comment by lintool** [comment]
hey @yuki617 - I'm still getting this test failure... can you please look into it? Thanks!

```
$ python -m unittest integrations.clprf.test_clprf.TestSearchIntegration.test_cross_validation
...

Traceback (most recent call last):
  File "./scripts/classifier_prf/cross_validate.py", line 143, in <module>
    generate_run_file(folders, clean_df, args.collection,
  File "./scripts/classifier_prf/cross_validate.py", line 95, in generate_run_file
    highest_alpha = train_df.iloc[-1, :].idxmax(axis=1)
  File "/cliphomes/jimmylin/.local/lib/python3.8/site-packages/pandas/core/series.py", line 2355, in idxmax
    i = self.argmax(axis, skipna, *args, **kwargs)
  File "/cliphomes/jimmylin/.local/lib/python3.8/site-packages/pandas/core/base.py", line 636, in argmax
    nv.validate_minmax_axis(axis)
  File "/cliphomes/jimmylin/.local/lib/python3.8/site-packages/pandas/compat/numpy/function.py", line 409, in validate_minmax_axis
    raise ValueError(f"`axis` must be fewer than the number of dimensions ({ndim})")
ValueError: `axis` must be fewer than the number of dimensions (1)
trec_eval.get_results: Cannot read results file './integrations/tmp2756/core17_lr.txt'
trec_eval: Quit in file './integrations/tmp2756/core17_lr.txt'
trec_eval.get_results: Cannot read results file './integrations/tmp2756/core17_lr.txt'
trec_eval: Quit in file './integrations/tmp2756/core17_lr.txt'


F
======================================================================
FAIL: test_cross_validation (integrations.clprf.test_clprf.TestSearchIntegration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/scratch2/jimmylin/pyserini/integrations/clprf/test_clprf.py", line 108, in test_cross_validation
    self.assertEqual(status, 0)
AssertionError: 512 != 0

----------------------------------------------------------------------
Ran 1 test in 4027.782s

FAILED (failures=1)
```

**Comment by yuki617** [comment]
Sorry, I went through all experiments for runs but missed the test for cross_validate. Here is the PR to fix it: https://github.com/castorini/pyserini/pull/958

**Comment by lintool** [comment]
Closed by #958
